src/toil/options/runner.py

```diff
@@ -0,0 +1,25 @@
+from argparse import ArgumentParser
+from toil.lib.conversions import human2bytes
+
+
+def add_runner_options(parser: ArgumentParser, cwl: bool = False, wdl: bool = False) -> None:
+    """
+    Add to the WDL or CWL runners options that are shared or the same between runners
+    :param parser: parser to add arguments to
+    :param cwl: bool
+    :param wdl: bool
+    :return: None
+    """
+    # This function should be constructed so that even when wdl and cwl are false, the "default" options are still added
+    run_imports_on_workers_arguments = ["--runImportsOnWorkers"]
+    if cwl:
+        run_imports_on_workers_arguments.append("--run-imports-on-workers")
+    parser.add_argument(*run_imports_on_workers_arguments, action="store_true", default=False, dest="run_imports_on_workers",
```

Comment:
I think we usually like to make new boolean options use `str2bool` and accept `False`/`True`, since we've been upgrading so many of the old ones to that format. Maybe we should upgrade this one too?